backend/Dockerfile

```diff
@@ -0,0 +1,5 @@
+FROM maven:3.8.5-openjdk-11
+WORKDIR /backend
+COPY . .
+RUN mvn clean install -Dmaven.test.skip=true
```

Comment:
Just a NOTE:  Create a task for maven profiles , to run this tests or not by separate different profiles and provide the profile here with -p <test_name>.. Don't change it now, just add a task as a Technical Dept

Reply:
I've made it into a task.
[https://htecgroup.atlassian.net/browse/SJQ2T-58](url)

I had to skip tests here since some tests don't really know they're inside a docker container and keep ignore evironment variables from `.env`.